prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -594,5 +595,20 @@ public async Task<ActionResult<AdjudicatorNote>> CreateSiteRegistrationNote(int
 
             return Ok(ApiResponse.Result(createdSiteRegistrationNote));
         }
+
+        // POST: api/Sites/remote-users
+        /// <summary>
+        /// Gets all of the Sites which have remote users who match college ID + licence num
+        /// </summary>
+        /// <param name="certifications"></param>
+        [HttpPost("/api/Sites/remote-users", Name = nameof(GetSitesByRemoteUserInfo))]
```

Comment:
this URL does not need to be absolute. it can just be "remote-users", since the controller is already located at "/api/sites"